### ISSUE TITLE
feature request: hiearchical package list

### ISSUE THREAD

**Issue by Davidbrcz** [Malabarba/paradox#163]
Hi,

Is is possible to display the list of packages not flat but in a hierarchical manner according to some regex/tagging in similar way to what's is done in org mode ?

For instance, all the packages for flycheck could be under a same entry, same goes for company, counsel, ...
That way, I would be easier to browse the list of packages and discover relevant packages.

**Comment by Malabarba** [comment]
It's something that could potentially be implemented, but no, it's not possible at the moment.

Unfortunately it would also be quite difficult, because the package buffer uses tabulated-list-mode for its structure.

**Comment by Davidbrcz** [comment]
Then some kind of virtual packages. Instead of displaying all of flycheck packages, simply an entry flychek-packages and pressing enter/what ever key would display a new buffer with all the packages in it.

**Comment by Malabarba** [comment]
That sounds very plausible, but it's still quite a delicate structural change. I think I'd be more comfortable if that were implemented as a separate package somehow.

**Comment by Davidbrcz** [comment]
Ok. I'll look into in when I'll have time

**Comment by a13** [comment]
There are keywords in package descriptions which you can click, flycheck is one of them

**Comment by a13** [comment]
>So I guess it's up to package maintainer (and maybe repo ?) to update their keywords.

yes, these are written in the code, so you always can make a pull request :)
